automation/src/main/java/CSS and Xpath

```diff
@@ -0,0 +1,21 @@
+1.
+	CSS:#userName
+	Xpath://input[@id='userName']
+2.
+	CSS:button.btn.btn-primary
+	Xpath://button
+3.
+	CSS:footer span
+	Xpath://div[@class='rt-td'] [text()='Vega']

```

Comment:
//div[@class='ReactTable -striped -highlight']//div[text()='Vega']

Spaces in xpath will throw an error